test/partials/accept-terms.spec.js

```diff
@@ -123,6 +124,27 @@ describe('accept-terms template', () => {
 		});
 	});
 
+	describe('transition', () => {
+		const params = {
+			isTransition: true
+		};
+
+		it('should have the default and transtion terms', () => {
+			const $ = context.template(params);
+			expectTerms($, { standard:1, transition: 3 });
+		});
+
+		it('should show immediate terms if transitionType immediate', () => {
+			const $ = context.template({...params, transitionType: 'immediate' });
+			expect($('label p.terms-transition--immediate').length).to.equal(1);
+		});
+
+		it('should show other terms if transitionType not immediate', () => {
+			const $ = context.template({...params, transitionType: 'endOfTerm' });
+			expect($('label p.terms-transition--other').length).to.equal(1);
```

Comment:
Assert without markup reference will not break the test on markup refactoring.
If you want to enforce that the test is covering also the markup probably it should be explicit.
```suggestion
			expect($('.terms-transition--other').length).to.equal(1);
```